storio-sqlite/src/test/java/com/pushtorefresh/storio2/sqlite/integration/NotifyAboutChangesTest.java

```diff
@@ -239,14 +246,15 @@ public void run() {
         // Steady!
         startAllThreadsLock.countDown(); // Go!
 
-        assertThat(allThreadsFinishedLock.await(20, SECONDS)).isTrue();
+        assertThat(allThreadsFinishedLock.await(25, SECONDS)).isTrue();
```

Comment:
Doesn't really matter, but since 30 seconds is now global limit for test duration, increasing this closer to 30 kinda makes sense :)